ServerCore/Pages/Threads/PuzzleThread.cshtml.cs

```diff
@@ -352,5 +370,38 @@ private async Task SendEmailNotifications(Message newMessage, Puzzle puzzle)
                 MailHelper.Singleton.SendPlaintextWithoutBcc(recipients.Select(r => r.Email), emailTitle, emailContent);
             }
         }
+
+        private ValidationResult IsMessageTextValid(string messageText)
+        {
+            if (string.IsNullOrWhiteSpace(messageText))
```

Comment:
Putting RequiredAttribute on the property would prevent this and add client-side validation